src/flair/count_sam_transcripts.py

```diff
@@ -96,10 +102,12 @@ def getannotinfo(args):
                 transcripttobpssindex[name] = bpindex
 
 
-    return transcripttoexons, transcripttobpssindex
+    return transcripttoexons, transcripttobpssindex, transcripttogenomicends
 
-def check_singleexon(read_start, read_end, tlen):
-    if read_start < 25 and read_end > tlen - 25:
+def check_singleexon(read_start, read_end, tlen, endnormdist):
+    # return True
+    # if read_start < 25 and read_end > tlen - 25:
```

Comment:
put in constants rather than hard-coding numbers.  This both makes the code hard to understand and hard to maintain with multiple places.  Pep8 uses all caps:

END_MIN_DIST = 25